lib/bb/configuration.rb

```diff
@@ -28,9 +28,9 @@ def connection
         conn.request :camelcase
         conn.request :json
         # Uncomment this out for generating curl output to send to MHV dev and test only
```

Comment:
You're right bout those lines showing PII if they're ever left in. That could be problematic, but perhaps I can wrap it in "if Rails.env.development?" such that when it is commented / uncommented it would still help mitigate that issue -- but I think what you did with respect to the reverse proxy is an even better solution. Perhaps that should be a good standard to follow for any new endpoints that have not been regressed in staging.